tests/coverage/await_ready.rs

```diff
@@ -0,0 +1,37 @@
+#![feature(coverage_attribute)]
+#![feature(custom_inner_attributes)] // for #![rustfmt::skip]
+#![feature(noop_waker)]
+#![rustfmt::skip]
```

Comment:
In `async.rs` this was to skip formatting of the whole file, but in this test we only need to skip formatting for `await_ready`, so we can put `#[rustfmt::skip]` directly on that function and remove `#![feature(custom_inner_attributes)]`.